spec/models/notification_spec.rb

```diff
@@ -5,23 +5,33 @@
     it 'can belong to an event' do
       expect(subject).to belong_to(:event)
     end
-
-    it 'has many recipients'
   end
-
   describe 'validations' do
     context 'status' do
       it 'accepts a valid status' do
-        notification = build(:notification, status: 'Scheduled')
+        allow_any_instance_of(Notification).to \
```

Comment:
Instead of this, could you instead make sure in the factory that each notification has at least on recipient?

Reply:
I prefer not to. You'll see in other codebases things in the Factory code like ->
    association :owner, factory: :person
But this makes the specs brittle, since you really don't care about the associated model. Also, you're now saving a lot more to the db, slowing tests. Imagine the person now has two more associations, and one of those has some too ! It can, and will, cascade.

Reply:
@chrisccerami Thanks for looking at this. If that answers your question, and you don't have other concerns, would you be willing to merge ? I would appreciate it.

Thanks

Reply:
I think instead of this, you could do something like:

```ruby
notification = build(:notification, departments: [build(:department)])
```

That way, nothing has to be persisted.

Reply:
I did not think that would work. I thought the department would need to be persisted to have an ID to put into the notification. I'll give it a try though !

Reply:
Yeah, the departments would need to be persisted in order to have an idea. However, your validation doesn't check for ids - only that `departments` is a non-nil value.

Reply:
@mxie 's idea worked. PR updated. Thanks !